test/e2e/edit-app-settings.spec.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs');
+const path = require('path');
+const { expect } = require('chai');
+const request = require('request-promise-native');
+
+const { getProjectUrl } = require('./cht-docker-utils');
+const {
+  DEFAULT_PROJECT_NAME,
```

Comment:
Looks like this field does not exist in cht-conf-utils?  (Tests now are just creating a project in `build/undefined`).

Reply:
where is typescript when we need it... 🤦‍♂️ good catch!